modules/ROOT/pages/log-points.adoc

```diff
@@ -25,21 +145,19 @@ Log points is supported on the following types of APIs:
 [NOTE]
 All basic endpoint APIs are displayed in the selectors regardless of whether or not the registered API has a supported Mule runtime engine version. If a log point is saved for a basic endpoint API with an unsupported version, the application policy fails.
```

Comment:
```suggestion
All basic endpoint APIs are displayed in the selectors regardless of whether the registered API has a supported Mule runtime engine version. If you save a log point for a basic endpoint API with an unsupported version, the application policy fails.
```

Reply:
I think that the previous text probably needs some rewording to be consistent with the text you have already changed. I did not try to do this, because I did not want to affect the tech accuracy.